rubin_sim/maf/metrics/brownDwarfMetric.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import healpy as hp
+from .baseMetric import BaseMetric
+import rubin_sim.maf.utils as mafUtils
+from scipy import interpolate
+
+
+class BDParallaxMetric(BaseMetric):
+    """Calculate the distance to which one could reach a parallax SNR for a given object
+    Modification of ParallaxMetric. Taken from:
+    https://github.com/jgizis/LSST-BD-Cadence/blob/main/bd_allLT_baseline_17.ipynb
```

Comment:
For more readability in the sphinx docs, this could be broken up a bit.. 
```Calculate the distance to which one could reach a given SNR in the parallax measurement for a given object. 

Modification of ParallaxMetric, illustrated in https://github.com/jgizis/LSST-BD-Cadence/blob/main/bd_allLT_baseline_17.ipynb

..